CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
Well, if we discover how to make anonymous link targets later, it's just a matter of change it anyway.